src/components/ApplicationLayout/AppStatus/AppStatus.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import classNames from "classnames";
+import type { HTMLProps, PropsWithChildren } from "react";
+
+import Panel from "components/Panel";
+
+type Props = PropsWithChildren & HTMLProps<HTMLDivElement>;
+
+const AppStatus = ({ children, className, ...props }: Props) => {
+  return (
+    <aside className={classNames("l-status", className)} {...props}>
+      <Panel wrapContent={false}>{children}</Panel>
```

Comment:
Maybe use bare children without the `Panel`, as it is not mandatory to have the panel here?